### ISSUE TITLE
Add default information displayed at the bottom

### ISSUE THREAD

**Issue by KashouC** [aseprite/aseprite#1874]
Okay the title is terrible but I'm not sure what to call it. The context bar at the bottom that shows how large your selection is, cursor coordinates, stuff like that. Right now it defaults to displaying nothing when the mouse is off screen or you have a window open, but I'd like it to always display basic information like image size, and when a selection is up show the selection size as well.

This is something that affects me a lot. Because VERY often I want to name files to include the canvas size, but I always forget to check the size before I open up the save as or export window, and then you can't find it anymore, so you have to close it, memorize the size, then open it up again.

Basically I'd like it to default to the "pen over canvas" context when it doesn't have anything else to show, possibly omitting the pen position.

**Comment by dacap** [comment]
Ok, so by default the status bar could show the "filename [ + selection size ] + sprite size" if there is a active sprite selected (or for example when we move the mouse over a sprite tab we can include this information).

**Comment by KashouC** [comment]
This is pretty much what you said, but I think the current information that's shown when you mouse over a sprite should be shown at all times for the current active sprite UNLESS there's something else, like mousing over another inactive sprite or what not, that should be displayed.

Except mouse position of course since we won't always be mousing over anything. I don't think I have any opinions on any further additions like filename or not. It might get annoying because you won't know where to expect the selection size and sprite size values to show up for example. It can already be kind of hard to find which value is which at a glance because it varies in position so much.
